assets/airports/cyhz.json

```diff
@@ -0,0 +1,657 @@
+{
+    "radio": {
+        "twr": "Halifax Tower",
+        "app": "Halifax Arrival",
+        "dep": "Halifax Departure"
+    },
+    "icao": "CYHZ",
+    "iata": "YHZ",
+    "magnetic_north": -18,
+    "ctr_radius": 80,
+    "ctr_ceiling": 12000,
+    "initial_alt": 5000,
+    "position": ["N44d52m52.73", "W063d30m33.62", "477.0ft"],
+    "rr_radius_nm": 5.0,
+    "rr_center": ["N44d52m47.00", "W063d30m37.00"],
+    "has_terrain": true,
+    "wind": {
+        "angle": 37,
+        "speed": 9
+    },
+    "arrivalRunway": "05",
+    "departureRunway": "05",
+    "airspace": [
+        {
+            "floor": 0,
+            "ceiling": 12000,
+            "airspace_class": "E",
```

Comment:
From what I can find, CYHZ might not have an approach control... it might actually just be handled by Moncton Center. The VFR chart seems to point toward a small Class C type area around the airport, but no designated approach control facility. 

It's kinda odd though, because I can see "Terminal" frequencies listed, which sounds like it could be an approach control, but Toronto _definitely_ has one, and their chart lists "APP" and "DEP" instead of "terminal", which makes me wonder what that really is for.

Do we have any way to verify what's going on with all this?

![image](https://cloud.githubusercontent.com/assets/5103735/26279882/9446c398-3d8e-11e7-8cba-51d40d30b6b6.png)

![image](https://cloud.githubusercontent.com/assets/5103735/26279883/a95ad332-3d8e-11e7-86b1-53495a54c407.png)